MapboxStatic/Color.swift

```diff
@@ -27,6 +27,19 @@ internal extension Color {
 
         return NSString(format: "%02x%02x%02x", Int(r), Int(g), Int(b)) as String
     }
+    
+    #if os(iOS)
```

Comment:
This condition causes the macOS, tvOS, and watchOS builds to fail. Both NSColor and UIColor have the same `getRed(_:green:blue:alpha:)` method, so conditionalizing this code on iOS should be unnecessary.